po/ru.po

```diff
@@ -5666,28 +5673,43 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"```rust,editable\n"
+"fn double(x: i32) -> i32 {\n"
+"    x + x\n"
+"}\n"
+"\n"
+"fn main() {\n"
+"    println!(\"удвоенное: {}\", double(7));\n"
+"}\n"
+"```"
 
 #: src/control-flow/blocks.md:38
 msgid "However if the last expression ends with `;`, then the resulting value and type is `()`."
 msgstr ""
+"Однако, если за последним выражением в блоке следует `;`, то результатом такого \n"
+"блока (а также его тип) будет `()`."
 
 #: src/control-flow/blocks.md:43
 msgid ""
 "* The point of this slide is to show that blocks have a type and value in Rust. \n"
 "* You can show how the value of the block changes by changing the last line in the block. For instance, adding/removing a semicolon or using a `return`.\n"
 "   "
 msgstr ""
+"* Смысл этого слайда показать, что в языке Rust у блоков есть значение и тип. \n"
+"* Можно показать, как меняется значение блока при изменении последней строки \n"
+"в блоке. Например, можно добавить/убрать точку с запятой или использовать `return`.\n"
+"   "
```

Comment:
```suggestion
```

Reply:
Not clear, are you suggesting to drop the content here?

Reply:
Yes, I think it's okay to drop this part, even if it's in the original.

Reply:
As long as you ensure that the `msgstr` and the `msgid` agree on the the use of a trailing `\n`. That is checked in CI via a call to `msgfmt` which will error out if it sees inconsistencies (it doesn't actually know about Markdown syntax, so this is just a very simple check).

Reply:
Oh, I see. Then it might be easier to leave it as it is.